rotkehlchen/chain/evm/decoding/aave/common.py

```diff
@@ -171,36 +207,78 @@ def _decode_deposit(
                 ):
                     event.event_subtype = HistoryEventSubType.RECEIVE_WRAPPED
                     resolved_asset = event.asset.resolve_to_asset_with_symbol()
-                    event.notes = f'Receive {event.balance.amount} {resolved_asset.symbol} from {self.label}'  # noqa: E501
+                    event.balance.amount = amount
```

Comment:
Didn't you say the amount should be minus the earned interest? So that we now split the events in 2?

EDIT: Looking at the v3 test changes you made it seems to be as you described. Though I somehow don't see it from the code.

Where is the amount coming from? Is it the deposit amount? Perhaps then rename the variable here (and in the other place I ask the same question) to make it obvious.

Reply:
yeah right this is the deposit amount, I'll rename the variable